assets/queries/ansible/aws/s3_bucket_allows_writeACP_action_from_all_principals/query.rego

```diff
@@ -0,0 +1,30 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  bucket := task["amazon.aws.s3_bucket"]
+  bucketName := task.name
+
+  bucket.policy.Statement[_].Effect == "Allow"
+  contains(lower(bucket.policy.Statement[_].Action), "write")
+  contains(lower(bucket.policy.Statement[_].Action), "acp")
+  bucket.policy.Statement[_].Principal == "*"
+
+    result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{amazon.aws.s3_bucket}}.policy.Statement", [bucketName]),
+                "issueType":        "IncorrectValue",
+                "keyExpectedValue": sprintf("amazon.aws.s3_bucket[%s] does not allow WriteACP Action From All Principals", [bucketName]),
+                "keyActualValue":   sprintf("amazon.aws.s3_bucket[%s] allows WriteACP Action From All Principals", [bucketName])
```

Comment:
the bucketName refers to the task name, not the name of the resource. For example, in the sample negative.yaml, bucketName refers to 'Bucket' and here you are targeting the resource name, which in that case is 'mys3bucket'.

Reply:
Mistake corrected.